src/translations/zu-ZA.json

```diff
@@ -1,65 +1,10 @@
-  "internationalization.src.localeDescriptor.en-IE": "Ireland",
-  "internationalization.src.localeDescriptor.en-US": "English (United States)",
-  "internationalization.src.localeDescriptor.es-419": "Spanish (Latin America & Caribbean)",
-  "internationalization.src.localeDescriptor.es-ES": "Spanish (Castilian)",
-  "internationalization.src.localeDescriptor.fi-FI": "Finnish",
-  "internationalization.src.localeDescriptor.fr": "French",
-  "internationalization.src.localeDescriptor.fr-BE": "Belgium",
-  "internationalization.src.localeDescriptor.fr-CA": "Quebec",
-  "internationalization.src.localeDescriptor.fr-FR": "French",
-  "internationalization.src.localeDescriptor.hi-IN": "Hindi",
-  "internationalization.src.localeDescriptor.hr-HR": "Croatian",
-  "internationalization.src.localeDescriptor.hu-HU": "Hungarian",
-  "internationalization.src.localeDescriptor.id-ID": "Indonesian",
-  "internationalization.src.localeDescriptor.it": "Italian",
-  "internationalization.src.localeDescriptor.it-IT": "Italian",
-  "internationalization.src.localeDescriptor.ja": "Japanese",
-  "internationalization.src.localeDescriptor.ja-JP": "Japanese",
-  "internationalization.src.localeDescriptor.ko-KR": "Korean",
-  "internationalization.src.localeDescriptor.lt-LT": "Lithuanian",
-  "internationalization.src.localeDescriptor.mr-IN": "Marathi",
-  "internationalization.src.localeDescriptor.ms-MY": "Malay",
-  "internationalization.src.localeDescriptor.nb-NO": "Norwegian Bokmål",
-  "internationalization.src.localeDescriptor.nl-NL": "Dutch",
-  "internationalization.src.localeDescriptor.pl-PL": "Polish",
-  "internationalization.src.localeDescriptor.pt-BR": "Portuguese (Brazil)",
-  "internationalization.src.localeDescriptor.pt-PT": "Portuguese (Portugal)",
-  "internationalization.src.localeDescriptor.ro-RO": "Romanian",
-  "internationalization.src.localeDescriptor.ru": "Russian",
-  "internationalization.src.localeDescriptor.ru-RU": "Russian",
-  "internationalization.src.localeDescriptor.sr-Latn-RS": "Latn-Serbian (Latin)",
-  "internationalization.src.localeDescriptor.sv-SE": "Swedish",
-  "internationalization.src.localeDescriptor.ta-IN": "Tamil",
-  "internationalization.src.localeDescriptor.th-TH": "Thai",
-  "internationalization.src.localeDescriptor.tr-TR": "Turkish",
-  "internationalization.src.localeDescriptor.uk-UA": "Ukrainian",
-  "internationalization.src.localeDescriptor.vi-VN": "Vietnamese",
-  "internationalization.src.localeDescriptor.zh-CN": "Chinese (Simplified)",
-  "internationalization.src.localeDescriptor.zh-HK": "Chinese (Traditional)",
-  "internationalization.src.localeDescriptor.zu-ZA": "Zulu",
   "ui.src.bottomMenu.moreChoicesButtonLabel": "Click to show more choices",
   "ui.src.bottomMenu.moreChoicesButtonPrimary": "More choices",
   "ui.src.bottomMenu.showPolicyButtonLabel": "Visit our privacy policy",
   "ui.src.bottomMenu.showPolicyButtonPrimary": "Our Privacy Policy",
```

Comment:
failed translation?

Reply:
ya some of these languages arent available in AWS translator service so they fall back to english for now. ive got a ticket to improve our intl process to support languages not in AWS translation, and also to have these translations go through human review. but its a matter of prioritization on when we would want to add that process in. it wont require much engineering work to support, its most us paying for some translation service and setting up a workflow for that service to translate these

Reply:
hiding these two in https://github.com/transcend-io/consent-manager-ui/pull/47